backend/berkeleytime/config/semesters/fall2023.py

```diff
@@ -0,0 +1,42 @@
+"""Configurations for Fall 2023."""
+from berkeleytime.config.finals.semesters.fall2023 import *
+
+import datetime
+
+CURRENT_SEMESTER = 'fall'
+CURRENT_YEAR = '2023'
+CURRENT_SEMESTER_DISPLAY = 'Fall 2023'
+
+# SIS API Keys
+SIS_TERM_ID = 2238
+
+TELEBEARS = {
+    'phase1_start': datetime.datetime(2023, 4, 17),
+    'phase1_end': datetime.datetime(2023, 6, 16),
+    'phase2_start': datetime.datetime(2023, 7, 17),
+    'phase2_end': datetime.datetime(2023, 8, 13),
+
+    'adj_start': datetime.datetime(2023, 8, 14),
+}
+
+INSTRUCTION = {
+    'instruction_start': datetime.datetime(2023, 8, 23, 00, 00),
+    'instruction_end': datetime.datetime(2023, 12, 8, 00, 00)
```

Comment:
Should be `date time.datetime(2023, 12, 1, 00, 00)`. From [here](https://registrar.berkeley.edu/Calendar/), the last day of formal instruction is Dec 1, 2023. In previous years, the registrar marked the last day of RRR week (in this year: Dec 8, 2023) as the last day of instruction, but it makes more sense to indicate it as currently marked in the registrar because there is no instruction during RRR.

Reply:
I think it's important to distinguish between the last day of *formal* instruction and the last day of (any) instruction. Some classes have review sessions, project presentations, or other instructional activities during RRR week, so RRR week is after the end of formal instruction but before the end of instruction. The last day of instruction isn't listed on the Google Calendar embedded on the registrar's page, but you can see it on the PDF version linked further down on the page. Based on the use of the last day of instruction (like, the end of RRR week) for `instruction_end` in previous terms, I believe it should refer to the end of (any) instruction. The date used for schedule export is different because the date is used for adding events to a calendar; regular class events don't generally occur during RRR week.

Reply:
That's a good point, i didn't know there were still ongoing classes during RRR. Good catch